projects/vdk-core/tests/functional/run/jobs/different-source-and-ingest-target/1_step.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-2024 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+from vdk.api.job_input import IJobInput
+
+
+def run(job_input: IJobInput):
```

Comment:
One more thing that should be possible and might be good idea to be covered for documentation purposes at least is following scenario.  

```
with job_input.get_managed_connection().cursor() as cursor: 
    cursor.execute("select * from stock") 
    job_input.send_tabular_data_for_ingestion(cursor,, [column_info[0] for column_info in cursor.description], ...)
```

Reply:
I will not cover it here since i change one of the values in the data for ingestion and this might make the code like this complicated. Maybe we can just write other tests for this scenario